internal/runners/use/use.go

```diff
@@ -112,3 +145,22 @@ func (u *Use) Run(params *Params) error {
 
 	return nil
 }
+
+func (u *Use) getLocalProjectPath(ns *project.Namespaced) string {
+	for namespace, paths := range projectfile.GetProjectMapping(u.config) {
+		if len(paths) == 0 {
+			continue
+		}
+		var namespaced project.Namespaced
+		err := namespaced.Set(namespace)
+		if err != nil {
+			logging.Debug("Cannot parse namespace: %v") // should not happen since this is stored
```

Comment:
nit: If we make this `logging.Critical` it should get reported to rollbar. An error happening here would mean we're doing something wrong with storing our project mapping.